source/index.html.md

```diff
@@ -3901,6 +3901,186 @@ This will once again allow you to receive funds from your customer via both DE a
 |400|[Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)|Bad Request (errors)|None|
 |404|[Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4)|Not Found|None|
 
+## Update a Receivable Contact
+
+<a id="opIdUpdateAReceivableContact"></a>
+
+> Code samples
+
+```shell
+curl --request PATCH \
+  --url https://api.sandbox.split.cash/contacts/2d2429c2-b868-455e-80ef-915df7c115a7/receivable/ \
+  --header 'authorization: Bearer {access-token}' \
+  --header 'content-type: application/json' \
+  --data '{"payid_name":"Bob Smith"}'
+```
+
+```ruby
+require 'uri'
+require 'net/http'
```

Comment:
Use double quotes for consistency? 

```suggestion
require "uri"
require "net/http"
```

Reply:
This and other changes to `source/index.html.md` are automatically generated by running `bundle exec foreman start`. This would mean if I modified the output file, any future changes to `split.yaml` would change my modifications back.